DESCRIPTION

```diff
@@ -17,7 +17,7 @@ Imports:
     devtools,
     httr
 Remotes:
-    annecori/EpiEstim,
+    annecori/EpiEstim@new-version,
```

Comment:
This will need changing back to `annecori/EpiEstim` (i.e. pointing at master) once the other PR has been merged.